tests/test_api_build.py

```diff
@@ -1170,6 +1167,8 @@ def test_unknown_selectors(testing_config):
     api.build(recipe, config=testing_config)
 
 
+# the locks can be very flaky on GitHub Windows Runners
+@pytest.mark.flaky(rerun=5, reruns_delay=2)
```

Comment:
Hmm... Given that GitHub Actions and Azure Pipelines are widely used by, e.g., conda-forge, this flakiness isn't unlikely to be encountered in production.
We should investigate this more (i.e., create issue/testing PR to track this).

Reply:
https://github.com/conda/conda-build/issues/4685